plugins/keyboard-navigation/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blockly/keyboard-navigation",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
I think this was already updated in a previous PR from 0.1 to 0.2, so I think we can just keep it at 0.2.0

Reply:
Done